spec/restful_resource/http_client_configuration_spec.rb

```diff
@@ -22,10 +22,6 @@ def find_middleware_args(adapter, name)
         expect(connection.adapter).to eq Faraday::Adapter::NetHttp
       end
 
-      it 'url_encodes requests' do
```

Comment:
As pointed out in the `FaradayMiddleware` gem here: https://github.com/lostisland/faraday_middleware/blob/b49e8ec17ab475056544fd330530f3e065b6d364/lib/faraday_middleware.rb#L48-L53

`FaradayMiddleware::EncodeJson` is now part of Faraday and not loaded when using 1.10.0 or above.